src/main/resources/config/liquibase/changelog/20200112214900_added_link_field_Outcome.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.2.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="SaraSilv" id="addLinkField">
+        <addColumn catalogName="outcome"
+                   schemaName= "public"
+                   tableName="outcome" >
+            <column afterColumn="id"
```

Comment:
Change to <column name="link" type="varchar(1000)"/>

afterColumn is used only if you do not want to add it at the end